LinearStage/v3/_init.yaml

```diff
@@ -9,7 +9,7 @@ serial_port: "/dev/ttyLinearStage1"
 steps_per_mm: 8000
 
 # For Igus
-stage_type: "Igus"
+# stage_type: "Igus"
```

Comment:
Please undo this change. The config schema says that Zaber is the default and the comment on line 11 says this is for Igus. (Also please consider deleting the comment: it clearly is for Igus if you have `stage_type: Igus` (you don't need the double quotes).
 
If you edit ts_LinearStage please make this property an enum, so that only the two values are actually allowed. Also please delete all default values in the schema: CSC config schemas are not supposed to have default values! (Though pluggable algorithm config schemas are allowed to have them, so they still exist, just in the CSC's main config schema).